### PR TITLE
FIX Set extraCodes to false for CMSProfileController

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-subsites/issues/535

### CODE REVIEW

File: src/Extensions/LeftAndMainSubsites.php [hunk 1]
```diff
@@ -88,7 +89,13 @@ public function sectionSites($includeMainSite = true, $mainSiteTitle = 'Main sit
         // Collect permissions - honour the LeftAndMain::required_permission_codes, current model requires
         // us to check if the user satisfies ALL permissions. Code partly copied from LeftAndMain::canView.
         $codes = [];
-        $extraCodes = Config::inst()->get(get_class($this->owner), 'required_permission_codes');
+        $ownerClass = get_class($this->owner);
```
Comment (michalkleiner) [inline src/Extensions/LeftAndMainSubsites.php:92]: You can also use `$ownerClass` on row 96 (old)/103 (new).

**Review by GuySartorelli — COMMENTED** [review]
Looks good and in a local kitchen sink install it passes the test that was failing.

I'll give you a chance to make the change that Michal has suggested, though that's not a deal breaker I think.

File: src/Extensions/LeftAndMainSubsites.php (hunk 1 above)
Comment (emteknetnz) [reply src/Extensions/LeftAndMainSubsites.php:92]: Upated

**Review by GuySartorelli — APPROVED** [review]
LGTM👍

**Pull request merged**